src/main/java/org/wildfly/security/auth/provider/FileSystemSecurityRealm.java

```diff
@@ -431,6 +435,15 @@ public void setAttributes(final Attributes attributes) throws RealmUnavailableEx
             replaceIdentity(newIdentity);
         }
 
+        @Override
+        public Attributes getAttributes() throws RealmUnavailableException {
+            final LoadedIdentity loadedIdentity = loadIdentity(false, false);
```

Comment:
The same problem - why you load attributes and credentials, while you use only attributes?